examples/demo3/src/backend/api/swagger.yaml

```diff
@@ -0,0 +1,227 @@
+openapi: 3.0.2
+info:
+  title: MEC Demo 3 API
+  description: This section describes how to use AdvantEdge mec services with a scenario
+    demonstrating use-case 1 - 3
```

Comment:
Demo 3 is an edge application that can be used with AdvantEDGE or ETSI MEC Sandbox to demonstrate MEC011 and MEC021 usage